### PR TITLE
update requests

### PR DESCRIPTION
**Pull request by peterromfeldhk**

### CODE REVIEW

File: shell.nix [hunk 1]
```diff
@@ -0,0 +1,14 @@
+with import <nixpkgs> {};
+
+mkShell {
+  buildInputs = [
+    nodejs-8_x
+    nodePackages_8_x.pnpm
+    nodePackages_8_x.yarn
```
Comment (lionello) [inline shell.nix]: I never use yarn tbh

Comment (peterromfeldhk) [reply shell.nix]: its used in the CI, so we either change the CI or use it :)

Comment (lionello) [reply shell.nix]: The CI doesn't care about shell.nix

Comment (peterromfeldhk) [reply shell.nix]: but you want to locally reproduce possible CI issues

**Pull request merged**